.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  # Enable version updates for nuget
+  - package-ecosystem: "nuget"
```

Comment:
will it update from MyGet or from NuGet? Do we want to keep up with the MyGet versions at all?

Reply:
i really don't know if that would update for myget. i will try in my repo and see if nuget is accepting myget/nuget

Reply:
@SergeyKanzhelev , with our latest release, i removed the myget feed from this repository, so it won't update from it. Let me know what do you think. Thank you.

Reply:
I think having both is good, especially during beta. Nuget is for formally released packages, myget is for bleeding edge daily builds.

Reply:
we were discussing if we will keep publishing every merge to myget. Probably that will in the next .net SIG